### DISCUSSION TITLE
Remove trailing zeros

### DISCUSSION THREAD

**Discussion by OliveiraPedro02** [astral-sh/ruff#13306] [Q&A] [answered]
Hi, is there a way to remove trailing zeros? 
For example
Before:
```
example_short_float = 1.0
example_long_float = 0001023.456000
example_int = 00001
```

After:
```
example_short_float = 1.
example_float = 1023.456
example_int = 1
```

Thanks in advance!

**Comment by MichaReiser** [comment, marked as answer]
Hi @OliveiraPedro02 

I did a quick check, and no, there isn't an existing rule for that. 


```python
example_int = 00001
```

This isn't a valid int literal.

**Reply by OliveiraPedro02** [reply]
Alright, thanks Micha!
